susemanager/src/mgr_bootstrap_data.py

```diff
@@ -212,7 +212,7 @@
     "salt-minion",
     "python3-apipkg*",
     "python3-iniconfig*",
-    "python3-looseversion",
```

Comment:
I think this is wrong. We only have one bootstrap repo definition across SLE 15 service packs. Supported SLE 15 service packs have Salt 3006.0 in their `basesystem` module (or via LTSS). Creating a bootstrap repo for e.g. SLE 15 SP4 requires `python3-looseversion`, it's not optional. 

What problem does this solve?